DOCS/man/options.rst

```diff
@@ -4051,6 +4051,10 @@ OSD
 
     See `Property Expansion`_.
 
+``--osd-playing-msg-duration=<time>``
+    Set the duration of ``osd-playing-msg`` in ms. Defaults to
+    ``osd-duration``.
```

Comment:
Minor wording nit. This technically defaults to 0 not ``osd-duration``. It should be noted that if this is unset, then the ``osd-playing-msg`` stays on screen for the duration of ``osd-duration``.